api/src/main/java/org/pmiops/workbench/config/ElasticSearchConfig.java

```diff
@@ -0,0 +1,29 @@
+package org.pmiops.workbench.config;
+
+import org.apache.http.HttpHost;
+import org.elasticsearch.client.RestClient;
+import org.elasticsearch.client.RestHighLevelClient;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import javax.inject.Provider;
+
+@Configuration
+public class ElasticSearchConfig {
+
+  @Autowired
+  private Provider<WorkbenchConfig> configProvider;
+
+  //Is Singleton scope appropriate here? or should we use another scope for elasticsearch?
```

Comment:
Docs indicate Singleton is correct here. Could you add this link to the doc here as well? 

https://www.elastic.co/guide/en/elasticsearch/client/java-rest/current/_changing_the_client_8217_s_initialization_code.html

Reply:
Added link.